units/recalculate_change_points.go

```diff
@@ -63,69 +66,60 @@ func makeMessage(msg string, id model.TimeSeriesId) message.Fields {
 
 func (j *RecalculateChangePointsJob) Run(ctx context.Context) {
 	defer j.MarkComplete()
-	conf := model.NewCedarConfig(j.env)
-	err := conf.Find()
-	if err != nil {
-		grip.Error(message.WrapError(err, message.Fields{
-			"message": "Unable to get cedar configuration",
-		}))
-		return
+	if j.conf == nil {
+		j.conf = model.NewCedarConfig(j.env)
 	}
-	if conf.Flags.DisableSignalProcessing == true {
+	if j.conf.Flags.DisableSignalProcessing == true {
 		grip.Info(makeMessage("signal processing is disabled, skipping processing", j.TimeSeriesId))
```

Comment:
in general we shouldn't log *every* time, just sometimes. 

`grip.InfoWhen(sometimes.Percent(10), makeMessage...)`